scripts/prettify_components.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Add this as a shell script to your WebStorm configurations
+# and as executable command to VS Code
+# Make sure the working directory for this command is equal to "<PROJECT_DIR>/src/components"
+
+# The path to the prettier config file.
+CONFIG_FILE="../../.prettierrc"
```

Comment:
Shouldn't this be just `.prettierrc`?
```suggestion
CONFIG_FILE=".prettierrc"
```